### PR TITLE
[config] Switch to new native-common repo for mesa and drivers

### PR DESCRIPTION
**Pull request by piggz**

### CODE REVIEW

**Comment by piggz** [comment]
One merged, we can drop mesa, qt and related builds from our repo. See https://build.merproject.org/project/show/nemo:devel:hw:native-common

File: sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[adaptation-community-native-common]
```
Comment (g7) [inline sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt]: File name should end with .ini

File: sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+repos = adaptation-community-native-common
+pattern = Feature adaptation community native common
+description = Common packages needed by community's native HW adaptation
+
+[repositories-release]
+adaptation-community-native-common = http://repo.merproject.org/obs/nemo:/@FLAVOUR@:/hw:/native-common/sailfish_latest_%(arch)/
```
Comment (g7) [inline sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt]: Perhaps I'm missing something, but how should we handle `@FLAVOUR@`? I think that the best course of action would be moving this ssu feature to the `community-adaptation` srcpackage, creating new packages `community-adaptation-native-common-{devel,testing,localbuild}` and then make droid-configs depend on the virtual package providing it.

That said, I'm fine with hardcoding `devel` for now.

@piggz @mlehtima thoughts?

File: sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt (hunk 1 above)
Comment (piggz) [reply sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt]: fixed

File: sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt (hunk 2 above)
Comment (piggz) [reply sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt]: lets go with devel for now, as thats the only options anyway?

File: sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt (hunk 1 above)
Comment (g7) [reply sparse/usr/share/ssu/features.d/adaptation-commnity-native-common.ini.txt]: Still shows as ini.txt

**Comment by g7** [comment]
LGTM :+1: , merging

**Comment by g7** [comment]
...for real

**Pull request merged**